elimu_yangu/settings.py

```diff
@@ -77,13 +77,42 @@
 }
 
 HURUMAP['comparative_levels'] = ["district", "region", "country"]
-HURUMAP['geometry_data'] = {
-    '2009': {
-                'country': 'geo/country.topojson',
-                'region': 'geo/region.topojson',
-                'district': 'geo/district.topojson'
-            }
+
+use_mapit = os.environ.get('USE_MAPIT', False)
+HURUMAP['USE_MAPIT'] = use_mapit
+if HURUMAP['USE_MAPIT'] == "True":
+  # use mapit settings
+  HURUMAP['geometry_data'] = {}
+  HURUMAP['mapit'] = {
+    'url': 'https://mapit.hurumap.org',
+    'country_code': 'TZ',
+    'generations': {
+        '2009': '1',
+        '2012': '1',
+        None: '1',  #  this should be based on the default_geo_version wazimap setting
+    },
+    'code_type': 'TZA',
```

Comment:
I'm just thinking out loud here @KhadijaMahanga but since we can specify countries in queries, shouldn't `code_type` represent something about where the `geojson` files come from? May be the organization/project/report etc.

This will mean say we want to view Tanzania in terms of police regions, `code_type` is what will change.

Reply:
@kilemensi we actually thought about that but we have no idea where the geojson files came from for most of the countries i.e. TZ, UG and KE and we're also not sure if there are government entities responsible for producing geojson files something like how there's always a government entity responsible for the census

Reply:
Not ideal @phillipahereza. This is something that you should prioritize and resolve as soon as possible. All data (including geojson files) in HURUmap **must** come from known and reputable sources.

But this shouldn't block this PR however. If you haven't, then you should create a story in PT to track this geojson issues.